### ISSUE TITLE
Verify compatibility (if ignite is not blocking) writing applications for Federated Learning Using PySyft

### ISSUE THREAD

**Issue by ramesht007** [pytorch/ignite#1378]

**Comment by ramesht007** [comment]
DIscussion needed...

**Comment by vfdev-5** [comment]
@ramesht007 thanks, this is according to one of project's Roadmap points : https://github.com/pytorch/ignite/wiki/Roadmap#integrations

Basic idea is to create a simple example application with PySyft and Ignite like ones for RL, GANs etc : https://github.com/pytorch/ignite/tree/master/examples

**Comment by ramesht007** [comment]
yup, this roadmap is great!.

**Comment by ramesht007** [comment]
So, In examples what exactly do you want?

**Comment by ramesht007** [comment]
Do you want a basic usage of ignite using pysyft or something else?

**Comment by vfdev-5** [comment]
Yes, a basic usage of ignite using pysyft and check if everything can work together for other more complex applications ...

**Comment by ramesht007** [comment]
ok, seems like a great idea. 😀🙂

**Comment by ramesht007** [comment]
@vfdev-5 can you assign this issue to me.

**Comment by vfdev-5** [comment]
Sure !

**Comment by ramesht007** [comment]
Thanks I will start working on this Issue. It might take some time as I might to look into Pysyft and Ignite compatibilty.

**Comment by vfdev-5** [comment]
Hi @ramesht007 any progress on this task ?

**Comment by ramesht007** [comment]
hey, @vfdev-5 
Pysyft is going into major refactor and the release is due in few weeks, 
So, I am waiting for the new release as it has much more functionality.
Thanks

**Comment by vfdev-5** [comment]
Hi @ramesht007 how about now on this task ?

**Comment by ramesht007** [comment]
Hi @vfdev-5 
sorry for the delayed reply, I had to shift countries as I was joining my master's program, Please give me some time I will be soon started working on this issue.

**Comment by sayantan1410** [comment]
@vfdev-5 Hey, I want to work on this issue, shall I ?

**Comment by vfdev-5** [comment]
@sayantan1410 go ahead
